assets/xml/objects/object_yb.xml

```diff
@@ -49,10 +49,10 @@
         <Limb Name="gYbPonyTailLimb" Type="Standard" EnumName="YB_LIMB_PONYTAIL" Offset="0x5E7C" />
         <Limb Name="gYbEmpty5E88Limb" Type="Standard" EnumName="YB_LIMB_EMPTY_5E88" Offset="0x5E88" />
```

Comment:
Perhaps add a todo to name these empty limbs based on player's skeleton when someone does that?